Sources/Exporters/DatadogExporter/DatadogExporter.swift

```diff
@@ -33,8 +33,12 @@ public class DatadogExporter: SpanExporter {
 
     public func export(spans: [SpanData]) -> SpanExporterResultCode {
         spans.forEach {
-            spansExporter.exportSpan(span: $0)
-            logsExporter.exportLogs(fromSpan: $0)
+            if $0.traceFlags.sampled || configuration.exportUnsampledSpans {
```

Comment:
I'm surprised sampled spans coming to exporter at all. Batching processor suppose to take care of filtering them out. I will LGTM anyway, but definitely something to look at going forward

Reply:
Yes, both BatchSpanProcessor and SimpleSpanProcessor filter unsampled spans by default, but you can configure them to not filter. I have a use case where I want to export the logs but not the spans where they were associated, and I can imagine other special cases were this configuration can be useful.